lldb/tools/lldb-dap/README.md

```diff
@@ -1,77 +1,30 @@
 # LLDB DAP
 
-## `lldb-dap` Configurations
+## Procuring the `lldb-dap` binary
 
-The extension requires the `lldb-dap` (formerly `lldb-vscode`) binary. It is a
-command line tool that implements the [Debug Adapter
-Protocol](https://microsoft.github.io/debug-adapter-protocol/). It is used to power the Visual Studio Code extension but can also be used with other IDEs and editors that support DAP.
-The protocol is easy to run remotely and also can allow other tools and IDEs to
-get a full featured debugger with a well defined protocol.
+The extension requires the `lldb-dap` (formerly `lldb-vscode`) binary.
+This binary is not currently packaged with the VS-Code extension. 
```

Comment:
Can we omit "currently" from that sentence? It leaves the impression that that might change in the future and when we discussed this in the RFC we said that this was a non-goal. Omitting it doesn't mean we'll never do it, but without a plan this just is just confusing.

Reply:
Nit: I've seen people write "VS Code" and "Visual Studio Code" but never "VS-Code". Can we consistently use the former?

Reply:
Fixed.

> It leaves the impression that that might change in the future 

I got ahead of myself while typing this...
See the RFC to also distribute/auto-download the `lldb-dap` binary here: https://discourse.llvm.org/t/rfc-automatically-download-lldb-dap/81322

Reply:
Ah, I had missed that!